tahrir_api/scripts/populatedb.py

```diff
@@ -0,0 +1,217 @@
+
+import datetime
+import os
+import random
+import sys
+import transaction
+
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from paste.deploy import appconfig
+from zope.sqlalchemy import ZopeTransactionExtension
+
+from ..model import (
+    Issuer,
+    Badge,
+    Person,
+    Team,
+    Milestone,
+    Series,
+    Assertion,
+    DeclarativeBase,
+)
+from ..utils import convert_name_to_id
+
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension(),
+                                        expire_on_commit=False))
```

Comment:
I would recommend lower casing this variable, since it is an instance of a Session and not a class itself.